### PR TITLE
ting

### PR DESCRIPTION
**Pull request by dluxe4444**

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -91,7 +50,18 @@ class Client {
                         }
                     }
                 })
-            }
+            },
+            delete(messageId = '@original') {
+                client.api.webhooks(clientID, interaction.token).messages(messageId).delete();
+            },
+            async edit(cont, messageId = '@original') {
```
Comment (foxydevelopment) [inline src/client.js:57]: I'd probably add a way to include components as well as that seems to work.

**Review by foxydevelopment — CHANGES_REQUESTED** [review]

File: src/client.js [hunk 2]
```diff
@@ -54,8 +54,8 @@ class Client {
             delete(messageId = '@original') {
                 client.api.webhooks(clientID, interaction.token).messages(messageId).delete();
             },
-            async edit(cont, messageId = '@original') {
-                const content = await createMessage(client, cont, interaction);
+            async edit(cont, messageId = '@original', options) {
+                const content = await createMessage(client, cont, interaction, options);
                 if (!input) {
```
Comment (foxydevelopment) [inline src/client.js:59]: This isn't defined. I'll attach a commit to fix that.

**Review by foxydevelopment — CHANGES_REQUESTED** [review]

**Review by foxydevelopment — APPROVED** [review]
Everything probably seems fine now, I'd probably wait to merge to see any last-minute bugs.

**Comment by dluxe4444** [comment]
I've already tested everything, so it should be fine to merge.

**Pull request merged**